phoneBehaviour/Call.java

```diff
@@ -0,0 +1,18 @@
+package phoneBehaviour;
+
+public class Call extends Contact {

```

Comment:
can you answer with is A for the following?
Call extends Contact?
No need here for this extension. Call class should be a simple class and Contact as an instance variable

```
public class CallFeature {
private Contact contact;

public CallFeature(Contact contact) {
this.contact = contact;
}...

}
```